### PR TITLE
Update with Decidim v0.25.2

### PR DESCRIPTION
**Pull request by laurajaime**
- Upgrade to Decidim v0.25.2
- Set Ruby version to 2.7.5
- Add first version
- Add CHANGELOG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,16 +83,21 @@ mount Decidim::FileAuthorizationHandler::AdminEngine => '/admin'
...
+cd spec/decidim_dummy_app/
+bundle exec rails decidim_file_authorization_handler:install:migrations
+RAILS_ENV=test bundle exec rails db:migrate
 ```
 
 And run tests:
 
 ```bash
-rspec spec
+bundle exec rspec spec
```
Comment (tramuntanal) [inline README.md:100]: :heart: for updating the doc!!

File: lib/decidim/file_authorization_handler/version.rb [hunk 2]
```diff
@@ -2,7 +2,12 @@
 
 module Decidim
   module FileAuthorizationHandler
-    # Uses the latest matching Decidim version
-    VERSION = "0.15.0"
+    def self.version
+      "0.1.0"
```
Comment (tramuntanal) [inline lib/decidim/file_authorization_handler/version.rb]: we can not go backwards when versioning... We moved the definition of the version, but the version in the gemspec should be coherent.

let's keep with the `# Uses the latest matching Decidim minor version` comment and upgrade to v `0.25.0`

**Review by tramuntanal — CHANGES_REQUESTED** [review]

File: lib/decidim/file_authorization_handler/version.rb (hunk 2 above)
Comment (laurajaime) [reply lib/decidim/file_authorization_handler/version.rb]: oh! ok, not problem :)

**Review by tramuntanal — APPROVED** [review]
LGTM!

Thanks for the ver change @LauraJaimeC

**Pull request merged**